greenpeace/planet4-master-theme
# PLANET-7269: Implement Sendgrid smtp integration

Ref: https://jira.greenpeace.org/browse/PLANET-6972
Ref: https://jira.greenpeace.org/browse/PLANET-7269

> Requirements
>    Add SENDGRID_API_KEY as a secret variable.
>    Use noreply@act.greenpeace.org as the sender.
>    For this phase of development let's mask this implementation with a flag under Planet 4 > Features and make it available only on dev sites.


## Test

On [test instance](https://www-dev.greenpeace.org/test-atlas/)
- activate feature in _Planet 4 > Features_
- change your account password, it should send an email "Password changed" to your inbox
